### PR TITLE
Fix #16386 : error on last modified invoices

### LINKED ISSUE #16386: last modified invoice in "last modified invoices" widget has a wrong status displayed when unpaid
# Bug
Status of unpaid invoices displayed in "last modified invoices" is wront (displays "payment started")
whereas it is correct in "last unpaid invoices" widget

## Environment
- **Version**: 13.0.0, 13.0.1
- **PHP**: 7.2
- **Database**: MariaDB 10
- **URL(s)**: /index.php

## Expected and actual behavior
[*Verbose description*]

## Steps to reproduce the behavior
create an invoice, leave it unpaid invoice and look at the widgets in the main dashboard

### PR DESCRIPTION
**Pull request by Hystepik**
Fix of a bug of display on box last modified invoices

### CODE REVIEW

File: htdocs/core/boxes/box_factures.php [hunk 1]
```diff
@@ -93,7 +93,8 @@ public function loadBox($max = 5)
 			$sql .= ", f.tva as total_tva";
 			$sql .= ", f.total_ttc";
 			$sql .= ", f.datef as df";
-			$sql .= ", f.paye, f.fk_statut as status, f.datec, f.tms";
+			$sql .= ", f.paye, f.fk_statut, f.datec, f.tms";
```
Comment (eldy) [inline htdocs/core/boxes/box_factures.php:96]: Warning, during your merge i think you lost a change done by another PR. For example
f.fk_statut was replaced with f.fk_statut as status 
this change must remain in your own PR.

File: htdocs/core/boxes/box_factures.php [hunk 2]
```diff
@@ -102,10 +103,13 @@ public function loadBox($max = 5)
 			$sql .= " FROM (".MAIN_DB_PREFIX."societe as s,".MAIN_DB_PREFIX."facture as f";
 			if (!$user->rights->societe->client->voir && !$user->socid) $sql .= ", ".MAIN_DB_PREFIX."societe_commerciaux as sc";
 			$sql .= ")";
+			$sql .= " LEFT JOIN ".MAIN_DB_PREFIX."paiement_facture as pf ON f.rowid=pf.fk_facture ";
 			$sql .= " WHERE f.fk_soc = s.rowid";
 			$sql .= " AND f.entity IN (".getEntity('invoice').")";
 			if (!$user->rights->societe->client->voir && !$user->socid) $sql .= " AND s.rowid = sc.fk_soc AND sc.fk_user = ".$user->id;
 			if ($user->socid)	$sql .= " AND s.rowid = ".$user->socid;
+			$sql .= " GROUP BY s.nom, s.rowid, s.email, s.code_client, s.logo, f.ref, f.date_lim_reglement,";
+			$sql .= " f.type, f.datef, f.total, f.tva, f.total_ttc, f.paye, f.fk_statut, f.rowid";
```
Comment (eldy) [inline htdocs/core/boxes/box_factures.php:112]: Can you check this page https://wiki.dolibarr.org/index.php?title=Language_and_development_rules#MySQL-MariaDB_specificity
And check you have enabled this option. When this option are on, it help to find bug into sql request because i think there is one error on your group by (list of fields must match all fields into select that are not calculation functions). With this option, you immediatly have a reported error that help to write cleaner sql.
Can you check.

**Comment by Hystepik** [comment]
it already has been fix so i close this PR
